bitnami/cassandra/README.md

```diff
@@ -225,31 +207,41 @@ You can enable TLS between client and server and between nodes. In order to do s
 
 In addition to this, you **must** create a secret containing the *keystore* and *truststore* certificates and their corresponding protection passwords. Then, set the `tlsEncryptionSecretName`  when deploying the chart.
 
-You can create the secret with this command assuming you have your certificates in your working directory (replace the PUT_YOUR_KEYSTORE_PASSWORD and PUT_YOUR_TRUSTSTORE_PASSWORD placeholders):
+You can create the secret (named for example `cassandra-tls`) using `--from-file=./keystore`, `--from-file=./truststore`, `--from-literal=keystore-password=PUT_YOUR_KEYSTORE_PASSWORD` and `--from-literal=truststore-password=PUT_YOUR_TRUSTSTORE_PASSWORD` options, assuming you have your certificates in your working directory (replace the PUT_YOUR_KEYSTORE_PASSWORD and PUT_YOUR_TRUSTSTORE_PASSWORD placeholders).To deploy Cassandra with TLS you can use those parameters:
 
 ```console
-kubectl create secret generic casssandra-tls --from-file=./keystore --from-file=./truststore --from-literal=keystore-password=PUT_YOUR_KEYSTORE_PASSWORD --from-literal=truststore-password=PUT_YOUR_TRUSTSTORE_PASSWORD
+cluster.internodeEncryption=all
+cluster.clientEncryption=true
+tlsEncryptionSecretName=cassandra-tls
 ```
 
-As an example of Cassandra installed with TLS you can use this command:
+### Initializing the database
+
+The [Bitnami cassandra](https://github.com/bitnami/bitnami-docker-cassandra) image allows having initialization scripts mounted in `/docker-entrypoint.initdb`. This is done in the chart by adding files in the `files/docker-entrypoint-initdb.d` folder (in order to do so, clone this chart) or by setting the `initDBConfigMap` value with a `ConfigMap` (named, for example, `init-db`) that includes the necessary `sh` or `cql` scripts:
 
 ```console
-helm install --name my-release bitnami/cassandra --set cluster.internodeEncryption=all \
-             --set cluster.clientEncryption=true --set tlsEncryptionSecretName=cassandra-tls \
+initDBConfigMap=init-db
 ```
 
-## Initializing the database
+### Using a custom Cassandra image
```

Comment:
I do not think we are adding valuable information for the user, we can remove this section though

Reply:
Makes sense, I just removed it